packages/core/src/forms/control/control.element.ts

```diff
@@ -128,8 +128,11 @@ export class CdsControl extends LitElement {
 
   @querySlot('label', {
     required: 'error',
-    requiredMessage: 'To meet a11y standards a <label> should be provided.',
+    requiredMessage: 'To meet a11y standards either a <label> or input[aria-label] should be provided.',
     assign: 'label',
+    exemptOn: _this => {
+      return _this.inputControl?.hasAttribute('aria-label');
+    },
```

Comment:
Great job with adding this to the decorator. I do think this lends to cleaner logic + testability. 👍